sdk/ml/azure-ai-ml/azure/ai/ml/entities/_load_functions.py

```diff
@@ -1065,3 +1066,31 @@ def load_feature_set_backfill_request(
         FeatureSetBackfillRequest,
         load_common(FeatureSetBackfillRequest, source, relative_origin, params_override, **kwargs),
     )
+
+
+def load_capability_host(
+    source: Union[str, PathLike, IO[AnyStr]],
+    *,
+    relative_origin: Optional[str] = None,
+    params_override: Optional[List[Dict]] = None,
+    **kwargs: Any,
+) -> CapabilityHost:
+    """Constructs a CapabilityHost object from a YAML file.
+
+    :param source: A path to a local YAML file or an already-open file object containing a capabilityhost configuration.
+        If the source is a path, it will be opened and read. If the source is an open file, the file will be read
+        directly.
+    :type source: Union[PathLike, str, io.TextIOWrapper]
+    :keyword relative_origin: The root directory for the YAML. This directory will be used as the origin for deducing
+        the relative locations of files referenced in the parsed YAML. Defaults to the same directory as source if
+        source is a file or file path input. Defaults to "./" if the source is a stream input with no name value.
+    :paramtype relative_origin: Optional[str]
+    :keyword params_override: Fields to overwrite on top of the yaml file.
+        Format is [{"field1": "value1"}, {"field2": "value2"}]
+    :paramtype params_override: List[Dict]
+    :raises ~azure.ai.ml.exceptions.ValidationException: Raised if CapabilityHost cannot be successfully validated.
+        Details will be provided in the error message.
+    :return: Loaded CapabilityHost object.
```

Comment:
can you add sample and example doc string?
for example look at load_job
 .. admonition:: Example:

        .. literalinclude:: ../samples/ml_samples_misc.py
            :start-after: [START load_job]
            :end-before: [END load_job]
            :language: python
            :dedent: 8
            :caption: Loading a Job from a YAML config file.
    """

Reply:
agreed and added